benchmarking/.gitignore

```diff
@@ -0,0 +1,2 @@
+/output
+Pipfile.lock
```

Comment:
I think we could add these to main .gitignore file, no need to add a new file here.

Reply:
Done, I also removed the Pipfile.lock because now only a requirements.txt file is used.